### PR TITLE
 CB-17935 "Service Unavailable" from CM APIs should likely use an exponential backoff retry policy

### PR DESCRIPTION
**Pull request by horadla23**
See detailed description in the commit message.

### CODE REVIEW

**Comment by lturcsanyi** [comment]
As we discussed privately, this is a static backoff retry policy based on the number of instances, which is fine but not aligned with JIRA. Please implement an exponential backoff strategy according to the ticket or rename the JIRA ticket/PR description/commit message, both would be fine with me.

File: cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/polling/task/AbstractClouderaManagerApiCheckerTask.java [hunk 1]
```diff
@@ -125,4 +127,11 @@ public void handleTimeout(T t) {
     public String successMessage(T t) {
         return String.format("Cloudera Manager API checking [%s] was a success", getPollingName());
     }
+
+    @Override
+    public long increasePollingBackoff(long defaultInterval, long consecutiveFailures) {
+        double newPollingBackoff = defaultInterval * Math.pow(EXPONENTIAL_BASE, consecutiveFailures);
```
Comment (bbihari) [inline cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/polling/task/AbstractClouderaManagerApiCheckerTask.java:133]: What is the biggest max-consecutive-failures parameter in CB?

**Comment by horadla23** [comment]
backoff values are in sec
Screenshot from 2022-08-02 16-26-14

**Comment by horadla23** [comment]
the 5th column is irrelevant on second thought, since polling will fail at the 5th failure without sleeping the thread again

**Review by lturcsanyi — APPROVED** [review]

**Pull request merged**